### PR TITLE
EHPR-151 | Remove erroroneous specialties, remove student

### PR DESCRIPTION
**Pull request by arranfw**
https://freshworks.atlassian.net/browse/EHPR-151

Ran the debugged script on the latest data sheet
- Student is removed as a stream + subspecialties
- Various erroneous specialties and subspecialties are fixed

### CODE REVIEW

File: packages/common/src/data/applicant_stream_data.json [hunk 1]
```diff
@@ -796,7 +716,7 @@
       "Renal": { "id": "Renal", "name": "Renal" },
       "RNFirstCall": { "id": "RNFirstCall", "name": "RN First Call" },
       "RuralRemote": { "id": "RuralRemote", "name": "Rural/Remote" },
-      "Wound": { "id": "Wound", "name": "Wound" },
+      "Wound": { "id": "Wound", "name": "\"Wound" },
```
Comment (fwpushan) [inline packages/common/src/data/applicant_stream_data.json:719]: I don't see any entry in the XLX file?

**Review by fwpushan — COMMENTED** [review]
LGTM. I need one clarification

File: packages/common/src/data/applicant_stream_data.json (hunk 1 above)
Comment (arranfw) [reply packages/common/src/data/applicant_stream_data.json:719]: Good catch, I'll look into it

Comment (arranfw) [reply packages/common/src/data/applicant_stream_data.json:719]: CSV!!! `Wound, Ostomy and Continence`

Comment (fwpushan) [reply packages/common/src/data/applicant_stream_data.json:719]: Are you updating the code ?

Comment (arranfw) [reply packages/common/src/data/applicant_stream_data.json:719]: Yeah I'll need to implement a CSV library

**Review by fwpushan — APPROVED** [review]

**Review by dbayly-freshworks — APPROVED** [review]
Looks ok. Very painful sounding changes.

**Pull request merged**